python/pyspark/sql/tests.py

```diff
@@ -3074,6 +3074,36 @@ def test_checking_csv_header(self):
         finally:
             shutil.rmtree(path)
 
+    def test_repr_html(self):
```

Comment:
This function only covers the most basic positive case. We need also add more test cases. For example, the results when `spark.sql.repl.eagerEval.enabled` is set to `false`.

Reply:
Got it, the follow up pr will enhance this test.